roles/installer/tasks/migrate_data.yml

```diff
@@ -59,14 +59,49 @@
     namespace: "{{ ansible_operator_meta.namespace }}"
     pod: "{{ postgres_pod_name }}"
     command: |
-      bash -c """
+      bash -c "
+      (while true; do echo 'Migrating data from old database...'; sleep 60; done) & keep_alive_pid=$!
       set -e -o pipefail
       PGPASSWORD='{{ awx_old_postgres_pass }}' {{ pgdump }} | PGPASSWORD='{{ awx_postgres_pass }}' {{ pg_restore }}
+      echo 'Killing keep alive process...'
+      kill $keep_alive_pid || true
+      wait
       echo 'Successful'
-      """
+      "
   no_log: "{{ no_log }}"
   register: data_migration
-  failed_when: "'Successful' not in data_migration.stdout"
+
+# Previous step result in defunct processes in the Postgres pod that need to be cleaned up
+# Restarting the Postgres pod will clean up the defunct processes
```

Comment:
This seems like it is inviting db inconsistencies if it gets killed at the wrong moment. If you insist on doing it this way, then I think it's important to make sure we gracefully shut down postgres before we kill the pod.

Reply:
awx deployments are scaled down before the data migration started 

there shouldn't be anything that's doing anything against postgres at this point so i think it would be safe to just kill the pod?